### DISCUSSION TITLE
syntax

### DISCUSSION THREAD

**Discussion by ko56** [shop-planner/shop3#99] [General]
Is there anything new w.r.t. the new syntax?

**Comment by rpgoldman** [comment]
I'm afraid not: I was on vacation for a week, and have been working on other things since then. It's on track, but not imminent, I'm afraid.

We are definitely going to go for a new top-level construct to replace `defdomain` so that we can have new, improved syntax, without breaking old domains and problems, but also without having the new parsing code be messed up by backward-compatibility with the old syntax.

**Reply by ko56** [reply]
I agree, a replacement for `defdomain` is the way to go, as I also said in my review.
